app/core/src/navigation/HotelsStack.js

```diff
@@ -70,9 +72,21 @@ export default {
           images,
         });
       }
+      function goToPayment(parameters) {
+        props.navigation.navigate('Payment', {
+          ...parameters,
+          // FIXME: we need to solve how to pass these data from search or map
```

Comment:
might I help?

Reply:
This is probably closely related to the #103 but I need to check it first. We will need to refactor this part completely because the idea is to create hotel package completely independent (with navigation and everything) so it will be possible to include just "Hotels" component in the Swift instead of implementing 4 components and navigation in Swift. So pushing it more to the core package is not good. I will start to dig deeper here very soon...